agent/clone_test.go

```diff
@@ -462,6 +462,28 @@ func testClone() {
 			return nil
 		}, 30*time.Second).Should(Succeed())
 
+		By("getting 500 when secret files doesn't exist")
```

Comment:
Why this test case run in this It block?
I think it is a different test case from the clone.